portfolio/src/main/java/com/google/sps/servlets/SentimentAnalysisServlet.java

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import com.google.cloud.language.v1.AnalyzeEntitiesRequest;
+import com.google.cloud.language.v1.AnalyzeEntitiesResponse;
+import com.google.cloud.language.v1.Document.Type;
+import com.google.cloud.language.v1.Entity;
+import com.google.cloud.language.v1.EncodingType;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+
+@WebServlet("/sentiment")
+public class SentimentAnalysisServlet extends HttpServlet {
+
+  public final static String SENTIMENT_AREA = "sentiment-comment";
+  String message;
+  String score;
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    message = request.getParameter(SENTIMENT_AREA);
+
+    // Use Language API to get Sentiment Score
+    Document doc = Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float tempScore = sentiment.getScore();
+    score = Float.toString(tempScore);
```

Comment:
IMO, `score = Float.toString(sentiment.getScore());` would still be readable while being more compact, but go with whichever you think is more readable. If you do have the temp variable, I think it should be named something like `floatScore` to more clearly contrast it with `score` (and maybe the latter should then be called `stringScore`, though I've seen plenty of instances where it would just be called `score`, and that seems fine too).